dataland-frontend/src/components/general/apiKey/CreateApiKeyCard.vue

```diff
@@ -52,7 +52,7 @@
           {{
```

Comment:
ok

Reply:
If we disable picking dates that are in the past (see my other comment on the Calendar-element to do so), we could switch the text here back to "Please select an expiration date", because then it will be impossible to pick a date that is in the past.

Reply:
done

Reply:
nice, thx 👍